### ISSUE TITLE
Auto-Type does not work on menu bar applications

### ISSUE THREAD

**Issue by sercanersoy** [keepassxreboot/keepassxc#5921]
## Overview

When I try to perform auto-type on forms of menu bar applications, KeePassXC performs the auto-typing on finder desktop screen (files starting with the typed letters get highlighted and the last highlighted file is being renamed after auto-type finally presses the enter key). To depict a concrete example: I use FortiClient and I can fill the form to connect to the VPN without opening the actual application, just from the menu bar as below.

![image](https://user-images.githubusercontent.com/32416586/104438495-1d069b80-55a1-11eb-8f80-27ca789547cb.png)

But the auto-type does not fill the form here, instead it does it to the Finder desktop as described above.

## Steps to Reproduce

1. Open a menu bar application form
2. Select the form field to perform auto-type
3. Open KeePassXC using Cmd+Tab
4. Select proper key and perform auto-type using Cmd+Shift+V

## Expected Behavior

KeePassXC should type the fields to the form of a menu bar application and hit enter.

## Actual Behavior

KeePassXC performs auto-type on Finder desktop, resulting in some files being highlighted because of the letters hit and the last highlighted file being renamed because of finally hitting the enter key.

## Context

I was using KeePassXC and successfully performing the auto-type on the menu bar application forms when my macOS version was Mojave. I updated it to the Big Sur and now I have the problem.

KeePassXC - Version 2.6.1
Revision: 9a35bba

Operating System: macOS Big Sur version 11.1
Desktop Env: ?
Windowing System: ?

**Comment by droidmonkey** [comment]
Try using global auto type

**Comment by sercanersoy** [comment]
I tried that too. Below are the entry and the window title:

![image](https://user-images.githubusercontent.com/32416586/104462164-446d6080-55c1-11eb-875c-b8169baf7623.png)

This is what I get after hitting my global auto-type shortcut:

![image](https://user-images.githubusercontent.com/32416586/104462254-62d35c00-55c1-11eb-9a68-f4867c314c2b.png)

It seems like this pop-up form is not interpreted as a 'window' by the operating system or KeePassXC do not see it. Anyway, I It would be great if I could use entry level auto-type also in Big Sur, which I was able to in macOS Mojave.
